bigbluebutton-tests/playwright/chat/characterLimit.js

```diff
@@ -0,0 +1,28 @@
+const { expect } = require('@playwright/test');
+const Page = require('../core/page');
+const { openChat } = require('./util');
+const e = require('../core/elements');
+
+class CharacterLimit extends Page {
+  constructor(browser, page) {
+    super(browser, page);
+  }
+
+  async test() {
+    await openChat(this.page);
+    const messageLocator = this.page.locator(e.chatUserMessageText);
+
+    await this.type(e.chatBox, e.longMessage5000);
+    await this.waitAndClick(e.sendButton);
+    await this.page.waitForSelector(e.chatUserMessageText);
```

Comment:
We can use the `this.waitForSelector` function here (and on lines 21 and 23), it also has a default timeout